vignettes/gDRcore.Rmd

```diff
@@ -65,7 +67,14 @@ Provided dataset needs to be merged into the one `data.table` object to be able
 `gDRimport::load_data()` and `gDRcore::merge_data()`.
 
 ```{r, nclude=FALSE}
-loaded_data <- gDRimport::load_data(gDRimport::manifest_path(td), gDRimport::template_path(td), gDRimport::result_path(td))
+loaded_data <-
+  suppressMessages(
```

Comment:
Is `suppressMessages` allowed in Bioc?

Reply:
Yes. It seems to be fine to use `supresssMessages()` in the vignette. We use `suppressMessages` in gDR and gDRutils already and there are no errors/warnings/notes related to this.